package.json

```diff
@@ -24,22 +26,27 @@
   },
   "homepage": "https://github.com/QubitProducts/d3-react-sparkline#readme",
   "devDependencies": {
-    "babel-core": "^5.8.22",
-    "babel-eslint": "^4.1.0",
-    "babel-loader": "^5.3.2",
-    "react-hot-loader": "^1.2.9",
-    "standard": "^5.1.0",
-    "webpack": "^1.12.0",
-    "webpack-dev-server": "^1.10.1",
-    "react": "^0.13.3",
-    "d3": "^3.5.6"
+    "babel-cli": "^6.4.5",
+    "babel-core": "^6.4.5",
+    "babel-eslint": "^5.0.0-beta6",
+    "babel-loader": "^6.2.1",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "d3": "^3.5.14",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7",
+    "react-hot-loader": "^1.3.0",
+    "standard": "^5.4.1",
+    "webpack": "^1.12.12",
+    "webpack-dev-server": "^1.14.1"
```

Comment:
Most dependencies could be updated, babel 5->6 wasn't hard. Just requires even more dependencies.